src/microbe_stage/Microbe.Contact.cs

```diff
@@ -337,9 +353,16 @@ public void Damage(float amount, string source)
         }
         else if (source == "pilus")
         {
+            if (invulnerabilityDuration > 0)
+                return;
+
             // Play the pilus sound
             PlaySoundEffect("res://assets/sounds/soundeffects/pilus_puncture_stab.ogg");
```

Comment:
Could you try adjusting the sound effect volume here to address the sound difference @buckly90 is hearing?

Reply:
Might be a good idea for someone to make sure it's not just something funky with audio mixing on my end just to be sure.

Don't want to accidentally blow out somebody's speakers with multiplied pili volume.

Reply:
I tried this myself and I have noticed an issue with the pili audio balance. in Master, the sound of a pilus hitting the player or another cell gets multiplied in volume because it gets played many times on the same couple frames. This makes it range from really quiet to really loud. Now, in my branch, it is only quiet. To boot, it needs quite a hefty volume boost to sound normal (to me, im not used to this game.) I tried 4.0 and it sounded okay. At least comparable to master.

Reply:
Did you push your changes? Once you've done that @buckly90 can probably listen to the changes again.

Reply:
Adjusted the volume and 4.0f sounds about right. Perhaps a tiny bit too loud, but even in an ocean full of spiky creatures I certainly wasn't being deafened or anything.